src/main/java/uk/gov/hmcts/darts/cases/mapper/CasesMapper.java

```diff
@@ -103,6 +106,18 @@ public SingleCase mapToSingleCase(CourtCaseEntity caseEntity) {
         return singleCase;
     }
 
+    public List<EventResponse> mapToEvents(List<EventEntity> eventEntities) {
+        return emptyIfNull(eventEntities).stream().map(this::mapToEvent).collect(Collectors.toList());
+    }
+
+    public EventResponse mapToEvent(EventEntity eventEntity) {
+        EventResponse event = new EventResponse();
+        event.setId(eventEntity.getId());
+        event.setTimestamp(eventEntity.getTimestamp().toLocalDate());
```

Comment:
This doesn't make sense for `timestamp` - the `toLocalDate()` will need to be removed when you update the openapi yaml to be `date-time` format